packages/template-retail-react-app/app/pages/product-list/index.jsx

```diff
@@ -545,6 +551,10 @@ const ProductList = (props) => {
                                                   product={productSearchItem}
                                                   enableFavourite={true}
                                                   isFavourite={isInWishlist}
+                                                  imageViewType={PRODUCT_TILE_IMAGE_VIEW_TYPE}
+                                                  selectableAttributeId={
+                                                      PRODUCT_TILE_SELECTABLE_ATTRIBUTE_ID
+                                                  }
```

Comment:
Was the intention to use a different set of constants for the product list?

```suggestion
                                                  imageViewType={PRODUCT_LIST_IMAGE_VIEW_TYPE}
                                                  selectableAttributeId={
                                                      PRODUCT_LIST_SELECTABLE_ATTRIBUTE_ID
                                                  }
```

Reply:
There was actually an unfinished thought there. In the PLP I was meant to use the `PRODUCT_LIST_IMAGE_VIEW_TYPE` and `PRODUCT_LIST_SELECTABLE_ATTRIBUTE_ID` in order to allow a granular configuration.. e.g. I want to change only the plp to use extra large images.. and then change the default product tile size to be medium across the rest of the site.